react-native-gutenberg-bridge/logger.js

```diff
@@ -0,0 +1,30 @@
+/** @format */
+
+import { NativeModules } from 'react-native';
+
+const { RNReactNativeGutenbergBridge } = NativeModules;
+
+const logger = {
+	debug: ( message ) => {
+		console.debug( message );
```

Comment:
Discussion points: Just in case I designed as executing console API internally but what do you think?